requirements.txt

```diff
@@ -1,17 +1,5 @@
-asn1crypto==0.24.0
-beautifulsoup4==4.8.0
-certifi==2019.9.11
-cffi==1.12.3
-chardet==3.0.4
-cryptography==2.7
-future==0.17.1
-idna==2.8
-lxml==4.4.1
-pycparser==2.19
-python-telegram-bot==12.1.1
-requests==2.22.0
-six==1.12.0
-soupsieve==1.9.4
-SQLAlchemy==1.3.8
-tornado==6.0.3
-urllib3==1.25.6
+sqlalchemy
+requests
+python-telegram-bot
+beautifulsoup4
+lxml
```

Comment:
I would prefer if these dependencies are pinned to a specific version. Particularly the telegram framework had some breaking changes in the past.